### ISSUE TITLE
Remove UTRs 

### ISSUE THREAD

**Issue by homonecloco** [EI-CoreBioinformatics/mikado#405]
I found another issue in the utilities. 

``` 
% remove_utrs.py -f gff3 TraesCS1A02G418200.gff3 TraesCS1A02G418200_no_utr.gff3
Traceback (most recent call last):
  File "/usr/local/bin/remove_utrs.py", line 13, in <module>
    def strip_utr(gene: Mikado.loci.Gene):
AttributeError: module 'Mikado' has no attribute 'loci'
```

**Comment by lucventurini** [comment]
Instead of
`import Mikado`

```python
import Mikado.loci
import Mikado.parsers
```

should solve the issue in that script.

**Comment by ljyanesm** [comment]
Fixed in 2.3.0
